src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -141,13 +219,32 @@ public TurretSubsystem(RobotStateManager robotStateManager) {
 
     highGearCANcoder.getConfigurator().apply(highGearSensorConfigs);
     lowGearCANcoder.getConfigurator().apply(lowGearSensorConfigs);
+    turretPIDController.setPID(
+        Constants.TurretConstants.TURRET_KP,
+        Constants.TurretConstants.TURRET_KI,
+        Constants.TurretConstants.TURRET_KD);
 
-    simEncoder =
+    simTurretEncoder =
         new SimDeviceSim(
             "CANEncoder:CANCoder (v6)", Constants.TurretConstants.highGearCAN_CODER_ID);
-    simTurretPos = simEncoder.getDouble("rawPositionInput");
+    simTurretPos = simTurretEncoder.getDouble("rawPositionInput");
 
-    turretPIDController.setIZone(Constants.TurretConstants.KIZ);
+    zeroTurret();
+    turretPIDController.setIZone(Constants.TurretConstants.TURRET_KIZ);
+
+    // Pitch
+    pitchPIDController =
+        new PIDController(
+            Constants.TurretConstants.PITCH_KP,
+            Constants.TurretConstants.PITCH_KI,
+            Constants.TurretConstants.PITCH_KD);
+    pitchCANcoder = new CANcoder(Constants.TurretConstants.PITCH_CANcoder_ID);
```

Comment:
Check with CAD to see if this is a cancoder, or a rev through bore encoder

Reply:
It's a rev through bore encoder. :|